r/configure

```diff
@@ -26,13 +26,14 @@
 # R CMD INSTALL --configure-vars='INCLUDE_DIR=/.../include LIB_DIR=/.../lib'
 
 # Library settings
-PKG_CONFIG_NAME="arrow parquet arrow-dataset"
+PKG_CONFIG_NAME="arrow"
 PKG_DEB_NAME="(unsuppored)"
 PKG_RPM_NAME="(unsuppored)"
 PKG_BREW_NAME="apache-arrow"
 PKG_TEST_HEADER="<arrow/api.h>"
-# These must be the same order as $(pkg-config --libs arrow-dataset)
-PKG_LIBS="-larrow_dataset -lparquet -larrow"
+PKG_LIBS="-larrow"
+PKG_DIRS=""
```

Comment:
Reading through the code, I'm not sure this is needed. I think in every case, you're defining $PKG_DIRS and never want to append an empty string to it.

Reply:
I confirmed this. Removed in edbfb7e3732cf63e63ba2a7380771a2c56e722f2